VideoLocker/src/main/java/org/edx/mobile/view/DiscussionAddCommentFragment.java

```diff
@@ -153,4 +154,18 @@ public void onException(Exception ex) {
         createCommentTask.setProgressDialog(createCommentProgressBar);
         createCommentTask.execute();
     }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+
+        if (getResources().getConfiguration().orientation == Configuration.ORIENTATION_LANDSCAPE) {
```

Comment:
Exactly what @BenjiLee said.

Plus, its really annoying to have the keyboard take all the screen, without the user knowing what the actual screen looks like and in which field are they inputting text.

Yeah, the hint is there but it doesn't give the whole context.